### PR TITLE
Fix system prune cmd user message with options

### PR DESCRIPTION
**Pull request by jmguzik**
[NO TESTS NEEDED]

Signed-off-by: Jakub Guzik <jakubmguzik@gmail.com>

/kind bug

1. Podman before fix (the same message):
```
$ podman system prune 

WARNING! This will remove:
        - all stopped containers
        - all stopped pods
        - all dangling images
        - all build cache
Are you sure you want to continue? [y/N] ^C
$ podman system prune -a

WARNING! This will remove:
        - all stopped containers
        - all stopped pods
        - all dangling images
        - all build cache
Are you sure you want to continue? [y/N] ^C
$ podman system prune -a --volumes 

WARNING! This will remove:
        - all stopped containers
        - all volumes not used by at least one container
        - all stopped pods
        - all dangling images
        - all build cache
Are you sure you want to continue? [y/N] ^C
$ podman system prune --volumes 

WARNING! This will remove:
        - all stopped containers
        - all volumes not used by at least one container
        - all stopped pods
        - all dangling images
        - all build cache
Are you sure you want to continue? [y/N] 

```
2. Docker:
```
$ docker system prune
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all dangling images
  - all dangling build cache

Are you sure you want to continue? [y/N] ^C
$ docker system prune -a
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all images without at least one container associated to them
  - all build cache

Are you sure you want to continue? [y/N] ^C
$ docker system prune -a --volumes 
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all volumes not used by at least one container
  - all images without at least one container associated to them
  - all build cache

Are you sure you want to continue? [y/N] ^C
$ docker system prune --volumes 
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all volumes not used by at least one container
  - all dangling images
  - all dangling build cache

Are you sure you want to continue? [y/N] ^C
```
3. podman after fix:
```
$ bin/podman system prune 
WARNING! This will remove:
        - all stopped containers
        - all networks not used by at least one container
        - all dangling images
        - all dangling build cache

Are you sure you want to continue? [y/N] ^C
$ bin/podman system prune -a
WARNING! This will remove:
        - all stopped containers
        - all networks not used by at least one container
        - all images without at least one container associated to them
        - all build cache

Are you sure you want to continue? [y/N] ^C
$ bin/podman system prune -a --volumes 
WARNING! This will remove:
        - all stopped containers
        - all networks not used by at least one container
        - all volumes not used by at least one container
        - all images without at least one container associated to them
        - all build cache

Are you sure you want to continue? [y/N] ^C
$ bin/podman system prune  --volumes   
WARNING! This will remove:
        - all stopped containers
        - all networks not used by at least one container
        - all volumes not used by at least one container
        - all dangling images
        - all dangling build cache

Are you sure you want to continue? [y/N] ^C
```

I kept original tabs in podman, but the message content is now the same as in docker.

### CODE REVIEW

File: cmd/podman/system/prune.go [hunk 1]
```diff
@@ -55,22 +55,18 @@ func init() {
...
-		fmt.Printf(`
-WARNING! This will remove:
-        - all stopped containers%s
-        - all stopped pods
-        - all dangling images
-        - all build cache
-Are you sure you want to continue? [y/N] `, volumeString)
+
+		message := fmt.Sprintf(createPruneWarningMessage(pruneOptions), volumeString, "Are you sure you want to continue? [y/N] ")
+		fmt.Printf("%s", message)
```
Comment (Luap99) [inline cmd/podman/system/prune.go:68]: Nit. You can use Printf directly. No need to call both Sprintf and Printf.

Comment (jmguzik) [reply cmd/podman/system/prune.go]: done

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**